src/core/table/table.jsx

```diff
@@ -0,0 +1,268 @@
+import React, { useState, useMemo } from 'react';
+import PropTypes from 'prop-types'
+import _ from 'lodash';
```

Comment:
If possible we could import functions individually, so that it can be tree-shaked

```suggestion
import { noop } from 'lodash';
```